code/controllers/subsystem/tgui.dm

```diff
@@ -216,30 +216,30 @@ SUBSYSTEM_DEF(tgui)
 		open_uis[src_object_key][ui.ui_key] = list() // Make a list for the ui_key.
 
 	// Append the UI to all the lists.
-	ui.user.open_tguis |= ui
+	ui.user.open_uis |= ui
 	var/list/uis = open_uis[src_object_key][ui.ui_key]
 	uis |= ui
 	processing_uis |= ui
 
- /**
-  * private
-  *
-  * Remove a UI from the list of open UIs.
-  *
-  * required ui datum/tgui The UI to be removed.
-  *
-  * return bool If the UI was removed or not.
- **/
+/**
+ * private
+ *
+ * Remove a UI from the list of open UIs.
+ *
+ * required ui datum/tgui The UI to be removed.
+ *
+ * return bool If the UI was removed or not.
+ */
 /datum/controller/subsystem/tgui/proc/on_close(datum/tgui/ui)
 	var/src_object_key = "[ui.src_object.UID()]"
 	if(isnull(open_uis[src_object_key]) || !istype(open_uis[src_object_key], /list))
-		return FALSE // It wasn't open.
+		return 0 // It wasn't open.
 	else if(isnull(open_uis[src_object_key][ui.ui_key]) || !istype(open_uis[src_object_key][ui.ui_key], /list))
-		return FALSE // It wasn't open.
+		return 0 // It wasn't open.
```

Comment:
Prob from copy pasting the existing TG code I recon?
```suggestion
		return FALSE // It wasn't open.
```

Reply:
I did see these when I was working on stuff, but I decided against changing them, since code continuity would be good. However, I am re-thinking the decision.